Source/Atomic/UI/UIListView.cpp

```diff
@@ -230,7 +230,6 @@ ListViewItemWidget::ListViewItemWidget(ListViewItem *item, ListViewItemSource *s
     SetLayoutDistribution(LAYOUT_DISTRIBUTION_GRAVITY);
     SetLayoutDistributionPosition(LAYOUT_DISTRIBUTION_POSITION_LEFT_TOP);
     SetPaintOverflowFadeout(false);
-    SetCapturing(false);
```

Comment:
This is needed to fix an issue when dragging an item up or down in the list view will cause the list to scroll